Projects/Core/StreamListener/Interface/StreamContinuation.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+
+public protocol StreamElement: Hashable { }
+
+public class StreamContinuation {
+  private let _yield: (any StreamType) -> Void
+
+  public init<T: StreamType>(_ continuation: AsyncStream<T>.Continuation) {
+    self._yield = { value in
+      guard let value = value as? T else { return }
+      continuation.yield(value)
+    }
+  }
+
+  public func yield(_ value: any StreamType) {
+    _yield(value)
+  }
+}
```

Comment:
finish 하는거도 추가하면 좋을거 같아!

```suggestion
public class StreamContinuation {
  private let _yield: (any StreamType) -> Void
  private let _finish: () -> Void

  public init<T: StreamType>(_ continuation: AsyncStream<T>.Continuation) {
    self._yield = { value in
      guard let value = value as? T else { return }
      continuation.yield(value)
    }
    self._finish = {
      continuation.finish()
    }
  }

  public func yield(_ value: any StreamType) {
    _yield(value)
  }
  
  public func finish() {
    _finish()
  }
}

```

Reply:
웅 피알내용에도 써놓긴 했는데 재시도 하면서 추가하려고 합니닷 , 재시도 스트림 생성 후에, api 다시 호출 되면 finish 하도록

Reply:
아 아냐 이것까지는 해당 피알에서 해야겟다 감삼다